bin/cliain/.dockerignore

```diff
@@ -0,0 +1,4 @@
+**/target
+target
+docker/data
+!target/release/cliain
```

Comment:
NIT: Maybe end the file with a newline, it's polite.

Reply:
done